README.md

```diff
@@ -14,30 +14,47 @@ detailed [API documentation](http://www.opesci.org/devito).
 
 ## Quickstart
 
-Devito can be installed from GitHub via pip:
+The recommended way to install Devito uses the Conda package manager
+for installation of the necessary software dependencies. Please
+install either [Anaconda](https://www.continuum.io/downloads) or
+[Miniconda](https://conda.io/miniconda.html) using the instructions
+provided at the download links.
+
+To install the editable development version Devito, including examples,
+tests and tutorial notebooks, please run the following commands:
 ```
-pip install --user git+https://github.com/opesci/devito.git
+git clone https://github.com/opesci/devito.git
+cd devito
+conda env create -f environment.yml
+source activate devito
+pip install -e .
 ```
 
-Alternatively Devito can be be installed manually from GitHub via:
+If you don't want to use the Conda environment manager, Devito can
+also be installed directly from GitHub via pip:
 ```
-git clone https://github.com/opesci/devito.git
-cd devito && pip install --user -r requirements.txt
+pip install --user git+https://github.com/opesci/devito.git
```

Comment:
Not necessarily, since `requirements.txt` is more of a "suggestion for a tested env configuration", while all the hard dependencies are in `setup.py`. The problem with the pure pip install is that our `requirements.txt` gets buried somewhere in the local Python install dir, so this is not that easy to predict or automate.